### ISSUE TITLE
Telegram notification doesn't work

### ISSUE THREAD

**Issue by mcbirne** [henrywhitaker3/Speedtest-Tracker#1150]
For about 2 to 3 weeks, the Telegram notification no longer works. I have restarted it several times and checked the settings. But unfortunately it doesn't work despite correct setup.

**Comment by tyokyo320** [comment]
I have the same problem....

**Comment by AlienPie** [comment]
This is not a bug, telegram notification work fine in v1.12.0 - I would check that your telegram details are correct and that other applications are able to forward using the same bot token and chat id.

**Comment by mcbirne** [comment]
Yes, other applications work great with the bot token and the chat ID. This application did until a few months ago. Currently I receive a Telegram message occasionally, although it should come daily.

**Comment by toupou** [comment]
same here. in the logs I can see "Telegram chat id and bot token unset" but they are set and saved. restart does not help either

**Comment by PetoMasaryk** [comment]
I have the same problem. When i check docker image logs, it does say that "Telegram chat id and bot token unset", despite i have this filled in application.

**Comment by AlienPie** [comment]
Have you tried configuring using the environment variable? TELEGRAM_BOT_TOKEN? This seems to work for me.

**Comment by PetoMasaryk** [comment]
Yes, i did just now but without any success.

**Comment by AlienPie** [comment]
:( I think at this point the project is abadoneded. I jumped ship to https://github.com/alexjustesen/speedtest-tracker which has been working well.
